dataset/src/integrationTest/kotlin/com/cosmotech/dataset/service/DatasetServiceIntegrationTest.kt

```diff
@@ -142,7 +142,7 @@ class DatasetServiceIntegrationTest : CsmRedisTestBase() {
 
   @BeforeEach
   fun beforeEach() {
-    every { getCurrentAccountIdentifier(any()) } returns CONNECTED_ADMIN_USER
+    every { getCurrentAccountIdentifier(any()) } returns TEST_USER_MAIL
```

Comment:
You swap user from admin to test, and every thing still work. i love this so much.
One day will have to make this straight.

Reply:
I wrote `makeDataset()` function and adapt other things also to make tests runs correctly after changing this :p

But I agree with you ;)